src/Player/Player.cpp

```diff
@@ -44,12 +43,13 @@ void Player::exeWait() {
     al::scaleVelocity(this, 0.7f);
 
     if (al::isPadTriggerA(mPort))
-        al::setNerve(this, &NrvPlayer.Jump);
+        return al::setNerve(this, &NrvPlayer.Jump);
     else if (al::isNearZero(al::getLeftStick(mPort), 0.001f))
-        al::setNerve(this, &NrvPlayer.Run);
+        return;
+    al::setNerve(this, &NrvPlayer.Run);
```

Comment:
Why would you change it like this?

Reply:
It was mismatching before but there was no comment. This makes the function match.

Reply:
`return XY` from `void` is not nice - try the original code with the second condition (`isNearZero`) flipped.